### ISSUE TITLE
Error when checking target: Expected activation_2 to have shape (6,6,512) but got array with shape (192,192,3)

### ISSUE THREAD

**Issue by CalinTimbusLucian** [keras-team/keras#9945]
Hello guys,
I've been working on a segmentation problem for many days and after finally finding out how to properly read the dataset, I ran into this problem: 
"Error when checking target: expected activation_9 to have shape (6, 6, 512) but got array with shape (192, 192, 3)".
I used the functional API, since I took the FCNN architecture from : 
https://github.com/divamgupta/image-segmentation-keras/blob/master/Models/FCN32.py
It is slightly modified and adapted in accordance with my task(IMAGE_ORDERING = "channels_last"(TensorFlow backend)  and the dimensions of the image 192X192).
Can anyone please help me?
Massive thanks in advance.
The architecture below is for FCNN, which I try to implement for the purpose of the segmentation.
![ph1](https://user-images.githubusercontent.com/36965748/38779921-d5f4a920-40d7-11e8-8e82-6689b121aaaa.PNG)
![ph2](https://user-images.githubusercontent.com/36965748/38779922-d60fa2ca-40d7-11e8-92a1-8def824fbd93.PNG)
![capture](https://user-images.githubusercontent.com/36965748/38779923-d62ee5c2-40d7-11e8-809f-df96b400d9f9.PNG)

**Comment by SalihaKevserKavuncu** [comment]
Eğittiğin model fotoğraf boyutunu aşıyor. Ya batch normalization eklemelisin veya conv katmanını azaltmalısın.
